python/dgllife/utils/complex_to_graph.py

```diff
@@ -140,27 +140,36 @@ def PN_graph_construction_and_featurization(ligand_mol,
         ligand_atom_indices_left = list(range(ligand_mol.GetNumAtoms()))
         protein_atom_indices_left = list(range(protein_mol.GetNumAtoms()))
 
-    # Construct bigraph for stage 1
-    node_featurizer = CanonicalAtomFeaturizer(atom_data_field='h')
-    edge_featurizer = CanonicalBondFeaturizer(bond_data_field='e')
+    # Node featurizer for stage 1
+    atoms = ['H','N','O','C','P','S','F','Br','Cl','I','Fe','Zn','Mg','Cs','Na','Mn','Ca','Co','Ni','Se','Cu','Cd','Hg','Sr','K']
+    atom_total_degrees = list(range(9))
+    atom_formal_charges = [-1, 0, 1]
+    atom_implicit_valence = list(range(5))
+    atom_explicit_valence = list(range(8))
+    atom_concat_featurizer = ConcatFeaturizer([partial(atom_type_one_hot, allowable_set=atoms), 
+                                               partial(atom_total_degree_one_hot, allowable_set=atom_total_degrees),
+                                               partial(atom_formal_charge_one_hot, allowable_set=atom_formal_charges),
+                                               atom_is_aromatic,
+                                               partial(atom_implicit_valence_one_hot, allowable_set=atom_implicit_valence),
+                                               partial(atom_explicit_valence_one_hot, allowable_set=atom_explicit_valence)])
```

Comment:
After `atom_formal_charge`, we need `atom_num_radical_electrons`, `atom_hybridization_one_hot`, `atom_is_aromatic`, and `atom_total_num_H_one_hot` in order. `CanonicalAtomFeaturizer` did not use `atom_explicit_valence_one_hot`.

Reply:
Due to the format of PDBBind, `atom_total_num_H_one_hot` yields the exact same features as `atom_implicit_valence_one_hot`. We need `atom_explicit_valence_one_hot` to represent the additional information for hydrogen atoms.